flash/core/model.py

```diff
@@ -630,3 +637,34 @@ def predict(self, inputs):
         composition = Composition(predict=comp)
         composition.serve(host=host, port=port)
         return composition
+
+    def to_package(
```

Comment:
@tchaton Is this something we may want to bring to lightning itself (similar to onnx/torchscript)?

Reply:
Yes, I think so. It doesn't work yet and Tom is sharing this is really early stage.

Reply:
Just a note, torch.package is in beta! We're still making changes and don't have BC guarantees on the APIs yet